src/app/users/store/users.effects.ts

```diff
@@ -1,8 +1,15 @@
 import { Injectable } from '@angular/core';
 import { Actions, createEffect, ofType } from '@ngrx/effects';
 import { EMPTY } from 'rxjs';
-import { map, mergeMap, catchError } from 'rxjs/operators';
+import {
+  map,
+  mergeMap,
+  catchError,
+  concatMap,
+  exhaustMap,
+} from 'rxjs/operators';
 import { UserService } from '../user.service';
+import { getAllUsersActions } from './users.actions';
 
 @Injectable()
 export class UserEffects {
```

Comment:
this looks like a mixture of stuff, this is maybe because of the lack of code distribution among files with a clear meaning.

Reply:
fixed thanks